src/styles/nav.scss

```
@@ -0,0 +1,79 @@
+@import "../utils/variables.scss";
+
+* {
+  margin: 0;
+}
+
+body {
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+}
+
+:root {
+  --color: $buttonAndActivesLinksColor;
+}
+
+.container {
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+}
+
+.logo {
+  display: block;
+  width: 40px;
+  height: 40px;
+}
+
+.headerList {
```

and move to nav.scss file
```suggestion
.nav__list {
```